### PR TITLE
Added npm auth token to local .npmrc file prior to publishing to npm registry

### PR DESCRIPTION
**Pull request by manicki**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -22,4 +22,6 @@ jobs:
     - name: Publish
       env:
         NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
-      run: npm publish --access public --@wmde:registry=https://registry.npmjs.org/
+      run:
```
Comment (wiese) [inline .github/workflows/release.yml]:
```suggestion
      run: |
```

Needs a pipe here to allow multi line

**Comment by wiese** [comment]
The env variables in the [example](https://github.com/vega/vega-lite/blob/7ba383fb2c8c2a8b5d1252c2061aaeef232484bb/.github/workflows/release.yml#L45) I unearthed are quite something - reading from `NPM_AUTH_TOKEN` and `NPM_TOKEN`, as well as setting `NODE_AUTH_TOKEN`. Shotgun debugging. It seems to work but we can't quite tell why.

File: .github/workflows/release.yml (hunk 1 above)
Comment (manicki) [reply .github/workflows/release.yml]: I had one job to copy two lines, and failed

Comment (wiese) [reply .github/workflows/release.yml]: > I had one job to copy two lines

And that's where you got it wrong. You have 3 lines to copy. :)

**Comment by manicki** [comment]
FWIW, `secrets.NPM_AUTH_TOKEN` is a name we control via github repo settings, so this could be unified easily.
why the vega example uses (apparently) two different token (or at least gives them two different names) is indeed fascinating.

**Review by wiese — APPROVED** [review]

**Comment by wiese** [comment]
(H)eureka.

```
npm info @wmde/vuex-helpers

@wmde/vuex-helpers@0.1.2 | GPL-2.0+ | deps: none | versions: 2

**Comment by manicki** [comment]
six pull requests later, mission accomplished: https://github.com/wmde/vuex-helpers/commit/1ec72eaed776a2f78b181f58dd414477479355b8/checks?check_suite_id=260957080 and https://www.npmjs.com/package/@wmde/vuex-helpers/v/0.1.2

thanks @wiese

**Pull request merged**